src/containers/UserProfile/AboutUser.js

```diff
@@ -0,0 +1,156 @@
+import React, { Component } from "react";
+import SimpleReactValidator from "simple-react-validator";
+import ReactNotification from "react-notifications-component";
+import "react-notifications-component/dist/theme.css";
+import addNotification from "../../utilities/index";
+import * as auth from "../../services/Session";
+import API from "../../api/index";
+
+class AboutUser extends Component {
+  constructor(props) {
+    super(props);
+
+    this.validator = new SimpleReactValidator();
+    this.state = {
+      about: this.props.aboutUser
+    };
+    this.handleUserInput = this.handleUserInput.bind(this);
+    this.notificationDOMRef = React.createRef();
+    this.submitForm = this.submitForm.bind(this);
+    this.onCancel = this.onCancel.bind(this);
+    this.modalRef = React.createRef();
+    this.formRef = null;
+  }
+
+  handleUserInput(e) {
+    const name = e.target.name;
+    const value = e.target.value;
+    this.setState({ [name]: value.trim() });
+  }
+
+  onCancel() {
+    this.formRef.reset();
+  }
+
+  submitForm(e) {
+    e.preventDefault();
+    if (this.validator.allValid()) {
+      var { about } = this.state;
```

Comment:
what this line doing?

Reply:
It is setting the state . var about = this.state.about